compressor/management/commands/compress.py

```diff
@@ -250,7 +250,15 @@ def compress(self, log=None, **options):
                     except Exception as e:
                         raise CommandError("An error occurred during rendering %s: "
                                            "%s" % (template.template_name, smart_text(e)))
-                    offline_manifest[key] = result
+
+                    if settings.COMPRESS_OFFLINE_USE_URL_PLACEHOLDER:
+                        # Replace settings.COMPRESS_URL with a placeholder
+                        offline_manifest[key] = result.replace(
+                            settings.COMPRESS_URL, settings.COMPRESS_URL_PLACEHOLDER
+                        )
+                    else:
+                        offline_manifest[key] = result
+
```

Comment:
_I've replied but now I've lost the reply, so I'll reply again_ :)

I agree, that's a controversial part. My idea was:

* `Command.compress()` returns full functional HTML code so you can use it
* when `manifest.json` contains _unprocessed_ (technical) code that should be _processed first_ before usage

If it's too controversial - I could try to simplify it.